src/precon.jl

```diff
@@ -4,15 +4,32 @@
 
 using ACE1
 
```

Comment:
I don't know how I missed this, or maybe it was even I who initiated this. But these are not preconditioners, they are priors. I think we need to discuss a big renaming here. Maybe not for this PR, but soon.

Reply:
Actually, should I just rename it? Is there anything that wouldn't just change from "precon" to "prior(s)", etc?

Reply:
should be prior or regularizer. I'm not 100% sure. What language do you think people in your grop would prefer?

Reply:
ok - I added some notes to the issue we can get some views before deciding.

Reply:
changed in the next commit